### PR TITLE
Add Copernicus Publications template

### PR DESCRIPTION
**Pull request by nuest**
This PR add the template by Copernicus Publications, a small open access publisher in the geosciences, based on https://publications.copernicus.org/for_authors/manuscript_preparation.html

### CODE REVIEW

File: R/copernicus_article.R [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+#' @export
+copernicus_article <- function(...,
+                         keep_tex         = TRUE,
+                         citation_package = "natbib",
+                         base_format = rmarkdown::pdf_document,
+                         md_extensions = c(
+                           "-autolink_bare_uris", # disables automatic links, needed for plain email in \correspondence
+                           "-auto_identifiers"    # disables \hypertarget commands
+                           )) {
+  if (inherits(base_format, "character")) {
```
Comment (yihui) [inline R/copernicus_article.R]: How about `is.character()`?

File: R/copernicus_article.R [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+#'
+#' Get the journal abbreviation for all journals supported by the copernicus article template.
+#'
+#' @param name A regular expression to filter the by the journal name, see \code{pattern} in \code{\link[base]{grep}}; defaults to \code{*}.
+#'
+#' @examples
+#' names(copernicus_journal_abbreviations())
+#' copernicus_journal_abbreviations(name = "Science Data")
+#'
+#' @export
```
Comment (yihui) [inline R/copernicus_article.R]: I'd merge the doc of this function with `copernicus_article`, i.e.

```r
#' @rdname copernicus_article
```

File: R/copernicus_article.R (hunk 1 above)
Comment (nuest) [reply R/copernicus_article.R]: Just copied this from `peerj_article`, but happy to change to `is.character()`.

File: R/copernicus_article.R (hunk 2 above)
Comment (nuest) [reply R/copernicus_article.R]: Done.

**Comment by nuest** [comment]
@yihui Thanks for taking a look, and my sincere apologies for the delay. I tried to incorporate your requirements. Could you please clarify what you mean by _"Could you also remove the standard LaTeX packages in this PR?"_

**Comment by nuest** [comment]
Contributor agreement is on the way, too.

**Comment by yihui** [comment]
I meant standard packages like `natbib.sty`. These could be easily installed from CTAN, so we should not ship copies of them in this R package.

To check if a package is on CTAN, you may use, e.g.,

```r
> tinytex:::parse_packages(files = 'natbib.sty')
tlmgr search --file --global '/natbib.sty'
[1] "natbib"
```

**Comment by nuest** [comment]
I understand. Removed `natbib.sty` and `pdfscreen.sty` from `inst/../copernicus_article/skeleton` but left `pdfsceencop.sty` in there.

```r
> tinytex:::parse_packages(files = 'pdfscreencop.sty')
tlmgr search --file --global '/pdfscreencop.sty'
character(0)
Warning message:
In tinytex:::parse_packages(files = "pdfscreencop.sty") :
  Failed to find a package that contains pdfscreencop.sty
```

**Comment by nuest** [comment]
The entry in the README.md was there already: https://github.com/rstudio/rticles/pull/172/files#diff-04c6e90faac2675aa89e2176d2eec7d8R55 Or would you like to have something different?

**Comment by nuest** [comment]
Yay 🎆 🎉  !

**Comment by nuest** [comment]
I've written a blog article planned to be published in the [EGU's GeoLog](https://blogs.egu.eu/geolog/), the academic society behind Copernicus Publications, along with some social media activity. The draft is at 

https://github.com/nuest/o2r-project.github.io/blob/egu-blog/_drafts/write-reproducible-manuscripts-for-copernicus-publishing-journals.md

Feedback of course welcome, but mostly just wanted to leave this here in case anyone stumbles across this PR.

**Comment by yihui** [comment]
Looks great! Thanks for sharing the post!

**Comment by yihui** [comment]
Please note that I just removed the `base_format` argument you copied from `peerj_article()`, since I noticed that people started doing this in other output formats (#208), which is really not necessary. To make use of **bookdown**'s features, now you need to use:

```yaml
output:
  bookdown::pdf_book:
    base_format: rticles::copernicus_article
```

**Comment by nuest** [comment]
Copernicus now starts advertising the template in their manuscript preparation guidelines, see e.g. journal ACP at https://www.atmospheric-chemistry-and-physics.net/for_authors/manuscript_preparation.html

image

**Comment by yihui** [comment]
@nuest Awesome! Thanks for letting us know!

**Pull request merged**